mbf_abstract_nav/src/mbf_abstract_nav/__init__.py

```diff
@@ -38,3 +38,5 @@ def add_mbf_abstract_nav_params(gen):
             "How long in seconds to allow for oscillation before executing recovery behaviors", 0.0, 0, 60)
     gen.add("oscillation_distance", double_t, 0,
             "How far in meters the robot must move to be considered not to be oscillating", 0.5, 0, 10)
+    gen.add("oscillation_angle", double_t, 0,
+            "How far in radians the robot must rotate to be considered not to be oscillating", math.pi, 0, 2*math.pi)
```

Comment:
You need to add `import math` at the top...

---

@corot Actually I don't know if this is better or worse than just typing `3.14` and `6.28` instead :joy: It kinda looks ugly on rqt :facepalm: 

![image](https://github.com/magazino/move_base_flex/assets/15384781/13b5c125-b436-46c7-a572-7b5fcfe02a59)

Reply:
I reverted the commit